app/controllers/api/v1/joke_getter_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::V1::JokeGetterController < ApplicationController
+  def index
+    headers = {
+      "x-rapidapi-host": "joke3.p.rapidapi.com",
+      "x-rapidapi-key": "285c319c04mshb039dd6377ad95cp13cb02jsnf523c698a5dd"
+    }
+    response = RestClient.get("https://joke3.p.rapidapi.com/v1/joke", headers)
+    parsed_response = JSON.parse(response.body)
```

Comment:
Please extract this functionality to a service as I demonstrated in my "Consume API" talk/demo

Reply:
Done aswell